docs/instances.md

```diff
@@ -4,10 +4,13 @@
 
 This page provides a list of `Websurfx` instances provided by us and our community. 
 
-|URL|Network|Version|Location|Behind Cloudflare?|Maintained By|TLS|IPv6|Comment|
+|URL|Network|Version|Location|Behind Cloudflare?|Status|TLS|IPv6|Comment|
```

Comment:
```suggestion
|URL|Network|Version|Location|Behind Cloudflare?|Uptime|Maintained By|TLS|IPv6|Comment|
```

I would suggest having the `maintained by` column as well. As this will allow in future when more instances will be added by other folks to provide their name or their project name. This is important as this will provide the users of these instances an idea of who is maintaining these instances and are they legitimate or privacy minded people or a well-known folk or project in the community. Which is as you know very critical when using something for privacy. Additionally, I would also, propose changing the column name status to uptime because I think it better reflects the purpose of the column I think. :slightly_smiling_face:

Reply:
The `Maintained By` column is just redundant, and I don't see its usage by any other project. Also, `Status` would be more fitting as it shows whether the website is up and running or not.

Reply:
> The `Maintained By` column is just redundant, and I don't see its usage by any other project. Also, `Status` would be more fitting as it shows whether the website is up and running or not.

Sorry for the late reply. 
Yes, you are right, and I agree, I have seen other projects not having a `maintained by` column. But I think having a `maintained by` column is important because of having the name of the folk with their contact information or the name of the project with its contact information. The person viewing this instances can check whether this is from a trustful person/project or not. This is similar to for example when you buy something from a market, you check whether this is a famous and trustful brand or not. If it is trustworthy, then you will buy it because you know you have trust on it. Let's say because they have a well established name, and they have been very consistent over the years with the product, let's say in quality, etc. And as discussed above, knowing whether who has hosted the project can be very critical when considering privacy and security. That's why I would strongly suggest having this column. :slightly_smiling_face: .

Reply:
Okay, then let's have the `Maintained By` column. Also, I think `Status` would be a more fitting name instead of `Uptime`.

Reply:
> Okay, then let's have the `Maintained By` column. Also, I think `Status` would be a more fitting name instead of `Uptime`.

Ok I see, then I would suggest just adding the `maintained by` column and filling it with the appropriate details. :slightly_smiling_face: